### ISSUE TITLE
Silent Hill 2 Remake Black Screen 

### ISSUE THREAD

**Issue by khalfo44** [jamesstringerparsec/Easy-GPU-PV#408]
I'm having issues running Silent Hill 2 on Hyper-V using Easy-GPU-PV script.

The game loads with Konami's splash screen at first and then it stops responding all of a sudden.

I've tried the following and still didn't reach to any solution yet:

- Changed the DirectX version from 12 to 11 on Steam using (-dx11) option.
- updated the GPU using the script.
- changed the game properties to run as administrator.

Still getting stuck on before the main menu loads.

I've also noticed the same behavior on Tekken 8 as well. I believe this issue is mainly happening on UE5 Games only.

If anyone had a similar issue and managed to solve it would really help me out.

Thanks.

**Comment by khalfo44** [comment]
Yeah, it's exactly the same behavior on Tekken 8 and I'm guessing on other UE5 games as well. It's super frustrating.

**Comment by StimzRx** [comment]
Possibly same issue as #396

**Comment by khalfo44** [comment]
The issue got fixed on its own after I updated the GPU driver files... Not sure if it's the update that fixed it or something else...

But now I'm facing with another challenge, I can see that the stream framerate is not accurate, It used to be constant 60FPS on almost all games, now I'm getting maximum 50fps which doesn't make any sense. I am using Moonlight with usbmmidd_v2 as a virtual usb monitor, also I'm using parsec as a secondary remote access, and both stream are not accurate where the it struggles to reach a constant 60FPS. 

I appreciate if anyone can guide me if you got this similar issue before and managed to fix it.
